src/main/java/Assignment6.java

```diff
@@ -0,0 +1,34 @@
+//Create a switch case with any 5 names of the company and their respective review.
+
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.InputStreamReader;
+
+public class Assignment6 {
+    public static void main(String[] args) throws IOException
```

Comment:
Instead of writing this code in void main, lets define a function for this. and lets create test in src/test/* package which tests your function